apps/block_scout_web/assets/js/pages/stakes.js

```diff
@@ -101,6 +102,7 @@ if ($stakesPage.length) {
   store.dispatch({ type: 'CHANNEL_CONNECTED', channel })
 
   channel.on('staking_update', msg => {
+    $('.tooltip').hide()
```

Comment:
I would rather limit the search by the top panel:
```suggestion
    $stakesTop.find('.tooltip').hide()
```

And separately handle tooltips in the pools table (probably, globally in the `async_listing_load.js`)

Reply:
Tooltip container out of stakesTop context and appear just before closed body tag.

Reply:
_okay.jpg_